core/templates/dev/head/pages/splash/splash.html

```diff
@@ -121,22 +121,21 @@ <h1 class="oppia-splash-h1" translate="I18N_SPLASH_TITLE"></h1>
         </div>
 
         <div class="oppia-splash-button-container">
-          {% if not user_is_logged_in %}
-            <!-- User must complete the registration process. -->
-            <button type="button"
-                    ng-click="onRedirectToLogin('/creator_dashboard?mode=create')"
-                    class="btn oppia-splash-button oppia-splash-button-create oppia-transition-200"
-                    style="left: -webkit-calc(130px + 15%); left: -moz-calc(130px + 15%); left: -o-calc(130px + 15%); left: calc(130px + 15%); bottom: 210px; z-index: 10;"
-                    translate="I18N_ACTION_CREATE_LESSON">
-            </button>
-          {% else %}
-            <button type="button"
-                    ng-click="onClickCreateExplorationButton()"
-                    class="btn oppia-splash-button oppia-splash-button-create oppia-transition-200"
-                    style="left: -webkit-calc(130px + 15%); left: -moz-calc(130px + 15%); left: -o-calc(130px + 15%); left: calc(130px + 15%); bottom: 210px; z-index: 10;"
-                    translate="I18N_ACTION_CREATE_LESSON">
-            </button>
-          {% endif %}
+          <!-- User must complete the registration process. -->
+          <button type="button"
+                  ng-if="!userIsLoggedIn"
+                  ng-click="onRedirectToLogin('/creator_dashboard?mode=create')"
+                  class="btn oppia-splash-button oppia-splash-button-create oppia-transition-200"
+                  style="left: -webkit-calc(130px + 15%); left: -moz-calc(130px + 15%); left: -o-calc(130px + 15%); left: calc(130px + 15%); bottom: 210px; z-index: 10;"
+                  translate="I18N_ACTION_CREATE_LESSON">
+          </button>
+          <button type="button"
+                  ng-if="userIsLoggedIn"
+                  ng-click="onClickCreateExplorationButton()"
+                  class="btn oppia-splash-button oppia-splash-button-create oppia-transition-200"
+                  style="left: -webkit-calc(130px + 15%); left: -moz-calc(130px + 15%); left: -o-calc(130px + 15%); left: calc(130px + 15%); bottom: 210px; z-index: 10;"
+                  translate="I18N_ACTION_CREATE_LESSON">
+          </button>
 
```

Comment:
Please remove this line.

Reply:
Sorry! which line? (Line no.)

Reply:
@DubeySandeep, oh sorry my bad, line no. 139, no need to leave a line between two button elements!

Reply:
It was earlier there so I left that as it was.

Reply:
Done! :)